### ISSUE TITLE
Can you please upload the pre-trained models?

### ISSUE THREAD

**Issue by kousik97** [flyingzhao/ExpressionNet#1]

**Comment by flyingzhao** [comment]
You can run it by yourself.

**Comment by kousik97** [comment]
How many epochs did u train to attain a test accuracy of 68%?

**Comment by flyingzhao** [comment]
Keras can stop and then start from the break point. So you can set a epoch number and watch the result until convergence.

**Comment by kousik97** [comment]
I know that, I am just trying to get an idea of how many hours I am supposed to train, so as to get an accuracy of 68%. Also, how much was your validation accuracy in the final epoch of training
